every.js

```diff
@@ -1,24 +1,23 @@
-const array = [1, 30, 39, 29, 10, 13];
 
-const every = (condition) => {
+const every = (array, condition) => {
 
     return array.reduce((accumulator, currentEl) => {
 
          let result = condition(currentEl);
+         let temp = accumulator;       
```

Comment:
you don't need to create this variable, you simply need to return true or false, and if accumulator already has value of false -> just return accumulator